### DISCUSSION TITLE
How to use fullTextIndex in a query?

### DISCUSSION THREAD

**Discussion by AnthonyLzq** [prisma/prisma#10877] [Q&A] [answered]
I have followed the [docs](https://www.prisma.io/docs/concepts/components/prisma-schema/indexes#index-configuration), but unfortunately there isn't any example with code about how to use it. I have already created a text index in my MongoDB database.

**Comment by austincrim** [comment, marked as answer]
Hey @AnthonyLzq 👋 ,

We do not currently support full text search commands directly through Prisma Client queries for Mongo or MySQL, as noted here on the docs page you linked:

> For now we do not enable the full text search commands in the Prisma Client. They will be implemented separately, and the progress can be followed in the [MongoDB](https://github.com/prisma/prisma/issues/9413) and [MySQL](https://github.com/prisma/prisma/issues/10415) issues.

But it's definitely coming! For now, we only support introspecting and migrating full text indexes in the schema file.

**Reply by AnthonyLzq** [reply]
Sorry, didn't notice that. Thanks.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
